### PR TITLE
Fix errors that occur when exporting transactions as csv - SER-1926

### PR DESCRIPTION
**Pull request by victorjbassey**
## Description

Backend PR - https://github.com/one-acre-fund/oaf--ph-ee-operations-app/pull/9

### CODE REVIEW

**Review by gilclefes — COMMENTED** [review]
@victorjbassey do we have release note for the repo, seems not updated

**Comment by victorjbassey** [comment]
> @victorjbassey do we have release note for the repo, seems not updated

@gilclefes I don't see a release note file for it. Will create one

**Review by gilclefes — APPROVED** [review]
looks good to me

File: src/app/payment-hub/request-to-pay/incoming-request-to-pay/incoming-request-to-pay.component.ts [hunk 1]
```diff
@@ -390,24 +390,8 @@ export class IncomingRequestToPayComponent implements OnInit {
...
-        {
-          responseType: "blob" as "json",
-          headers: new HttpHeaders().append("Content-Type", "application/json"),
-        }
-      )
-      .subscribe((val) => {
-        console.log("POST call successful value returned in body", val);
-        this.downLoadFile(val, "application/csv");
-      });
+    filterBy[filterBy.cars] = filterBy.val;
```
Comment (shejasabin) [inline src/app/payment-hub/request-to-pay/incoming-request-to-pay/incoming-request-to-pay.component.ts:393]: @victorjbassey This is not changed to status ?

Comment (victorjbassey) [reply src/app/payment-hub/request-to-pay/incoming-request-to-pay/incoming-request-to-pay.component.ts:393]: No, it's not necessary here

Comment (shejasabin) [reply src/app/payment-hub/request-to-pay/incoming-request-to-pay/incoming-request-to-pay.component.ts:393]: okay

**Pull request merged**